repoaudit/repoaudit/__init__.py

```diff
@@ -62,30 +94,21 @@ def main() -> None:
 @recursive_option
 @click.argument("url")
 @click.option("--dists", help="Comma separated list of distributions.")
+@apt_sources_option
 @file_option
 @pubkey_option
-def apt(recursive: bool, url: str, dists: str, output: str, pubkeys: str) -> None:
+def apt(recursive: bool, url: str, dists: str, apt_source: str, output: str, pubkeys: str) -> None:
     """Validate an apt repository at URL."""
-    if recursive:
-        urls = get_repo_urls(url)
+    if apt_source:
+        with open(apt_source, "r") as f:
+            lines = [line.strip() for line in f if line.startswith("deb")]
+            for line in lines:
+                fields = line.split(" ")
+                _url = fields[1]
```

Comment:
I'm curious about why you're using `_url` vs `url`?

Reply:
I had thought it would affect the initial arguments but now that I think about it, there's no reason for me to do that. Thanks for pointing this out!